docs/sample_config.yaml

```diff
@@ -409,6 +409,14 @@ retention:
   #    longest_max_lifetime: 1y
   #    interval: 1d
 
+# Inhibits errors related to the existence or lack thereof (depending on the
+# endpoint) on the /requestToken endpoints. Instead, the homeserver will respond
+# with a 200 status code and a fake session ID ('sid').
```

Comment:
I find this first sentence really hard to understand, maybe something like the following is clearer:

> Inhibits the /requestToken endpoints from returning an error, which might leak information about whether an e-mail address is in use or not. Note that for some endpoints the error situation is the e-mail already being used, and for others the error is entering the e-mail being unused.

Reply:
Done in https://github.com/matrix-org/synapse/pull/7315/commits/844309dd78bcd6dbd0b1699dc4c18aeae0396d24